contactsummary/js/test/services/api_test.js

```diff
@@ -27,7 +27,14 @@ define([
             var expectedResult = {data: true}, actualResult = {};
 
             // Set up response from HTTP backend
-            httpBackend.expectPOST('/civicrm/ajax/rest').respond(expectedResult);
+            httpBackend.expectGET('/civicrm/ajax/rest?action=get&entity=test&json=1&rowCount=0&sequential=1').respond(expectedResult);
```

Comment:
The test was failing because the service to get information from the backend through the API was implemented using GET instead of POST.

https://github.com/civicrm/civihr/blob/ed0da6aecb7f1a140f8a40398aac8e1a48d356e6/contactsummary/js/src/contact-summary/services/api.js#L76